INSTALL.md

```diff
@@ -2,24 +2,25 @@
 
 Helm charts are provided inside https://github.com/eclipse-tractusx/sd-factory
 
-1.) Using helm commands: <br />
+1.) Installation from released chart: <br />
 
 How to install application using helm:  <br />
     helm install ReleaseName ChartName
     
-    a.) Add helm repository in tractusx:
-           helm repo add sd-factory https://eclipse-tractusx.github.io/charts/dev
+    a.) Add helm repository for sdfactory:
+           helm repo add [name] https://eclipse-tractusx.github.io/charts/dev
     b.) To search the specific repo in helm repositories 
-           helm search repo sd-factory/sdfactory
+           helm search repo [name]/sdfactory
     c.) To install using helm command:
-           helm install sdf sd-factory/sdfactory
+           helm install [ReleaseName] [name]/sdfactory
```

Comment:
same as in the 2.e, this command installs on a k8s cluster; maybe mention that.
also, in 2.e you mention a namespace, not sure why here not as well.

Reply:
This has been taken care